pom.xml

```diff
@@ -280,6 +281,12 @@
 
   <profiles>
     <profile>
+      <id>spark_core-2.10</id>
+      <properties>
+        <spark.core.artifactid>spark_core_2.10</spark.core.artifactid>
+      </properties>
+    </profile>
+    <profile>
```

Comment:
How are Protobuf files handled in this profile? The other 2 profiles deal with scenarios where you wish to re-generate protobuf files or re-use the ones distributed in the repo. If this new profile is activated, what happens?